include/mega/node.h

```diff
@@ -89,6 +89,28 @@ struct MEGA_API PublicLink
     bool isExpired();
 };
 
+// Container storing FileFingerprint* (Node* in practice) ordered by fingerprint.
+struct Fingerprints
+{
+    // maps FileFingerprints to node
+    using fingerprint_set = std::multiset<FileFingerprint*, FileFingerprintCmp>;
+    using iterator = fingerprint_set::iterator;
+
+    void newnode(Node* n);
+    void add(Node* n);
+    void remove(Node* n);
+    void clear();
+    m_off_t getSumSizes();
+
+    Node* nodebyfingerprint(FileFingerprint* fingerprint);
+    node_vector *nodesbyfingerprint(FileFingerprint* fingerprint);
+
+private:
+    fingerprint_set m_fingerprints;
```

Comment:
Please, refactor this variable and the one below to the code style: `mFingerprints` and `mSumSizes`. Thanks

Reply:
done :-)